change/@microsoft-teams-js-e23ac78a-fff1-4ea1-919e-9b9e06ad3af7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "gitmagic additions to 2.0-preview for February 28, 2022. Changes merged in from V1 include stageView implementation and modifications to tasks",
```

Comment:
Since this is the only thing that people will see in the changelog about this change, I was hoping we would be super specific. Something like: (sorry, I should have included an example the first time around):

> ...Changes merged in from V1 include stageView implementation and modifying `dialog.resize()` and `dialog.submit()` to also work in FrameContexts `sidePanel`, `content`, `meetingStage`, and `task` rather than just `task`.

Reply:
Got it, going forward we want to be super specific. I didn't know the level of specificity we were aiming for, I'll make that change now

Reply:
Yeah IMO it's a bit counterintuitive that we'd want it to be so specific considering it's only done on one line and you can't really use bullet points. Hopefully we can come up with something better for documenting on the changelog if this turns out to be a big issue.